grunt/clean.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
     images: 'dist/img',
     less: 'dist/less',
     licenses: 'dist/Licenses',
-    e2e: ['e2e/dist', 'e2e/_test-output', 'e2e/coverage'],
+    e2e: ['e2e/dist', 'e2e/_test-output', 'e2e/coverage', 'e2e/html', 'e2e/xml'],
     shim: 'dist/dist',
-    bower: 'target/bower'
+    bower: 'target/bower',
+    webdriver: 'node_modules/protractor/node_modules/webdriver-manager/selenium/chromedriver_*.zip'
```

Comment:
Can we just copy the driver into e2e/dist and use it from there? And have it cleaned up with everything else? I don't really like the idea of adding/removing files in node_modules apart from via npm.